src/screens/ManageTag/ManageTag.tsx

```
@@ -282,23 +288,23 @@ function ManageTag(): JSX.Element {
+            Error occurred while loading assigned users
           </h6>
         </div>
       </div>
     );
   }
 
   const userTagAssignedMembers =
-    userTagAssignedMembersData?.getAssignedUsers.usersAssignedTo.edges.map(
-      (edge) => edge.node,
+    userTagAssignedMembersData?.tag.assignees.edges.map(
+      (edge: any) => edge.node,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)